sync/sync_test.go

```diff
@@ -142,7 +137,6 @@ func TestSyncBlocks(t *testing.T) {
 }
 
 func TestReorg(t *testing.T) {
-	t.Parallel()
```

Comment:
@IronGauntlets if I leave `t.Parallel()` here and in `TestSyncBlocks` (top level) it leads from time to time to bugs (idk why :( )

Reply:
I'm happy for you to remove it. I am not sure why this happens.